### DISCUSSION TITLE
Cookie consent module

### DISCUSSION THREAD

**Discussion by jayachandraoggy** [breezefront/community#137] [3. Integrations]
Hello,

We need a cookie consent module, something similar to this module: https://github.com/phpro/phpro-mage2-module-cookie-consent/ but it is not working in the Breeze theme. Please check and I saw there are 3 JS files in the module.

**Comment by vovayatsyuk** [comment]
In case you are looking at other similar solutions too, we have our own module and it works with Breeze: https://swissuplabs.com/m2-gdpr-extension.html

**Reply by jayachandraoggy** [reply]
Hello @vovayatsyuk ,

Thanks for suggesting the similar solutions. But anyway I made the [(Phppro module compatible)](https://github.com/phpro/phpro-mage2-module-cookie-consent/pull/13) with the Breezefront them.

**Reply by vovayatsyuk** [reply]
That's awesome!
